apps/backend/src/modules/friends/friend.repository.ts

```diff
@@ -121,6 +121,23 @@ class FriendRepository implements Repository<UserEntity> {
 		});
 	}
 
+	public async getIsMutualFollowersByIds(
```

Comment:
```suggestion
	public async checkIsMutualFollowersByIds(
```
methods that return boolean should start with `check<is | has>`